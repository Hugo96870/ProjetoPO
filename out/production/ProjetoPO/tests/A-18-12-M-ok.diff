*** tests/A-18-12-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-12-M-ok.outhyp	2020-12-04 11:15:13.944600800 +0000
***************
*** 47,53 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|300|270
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
--- 47,53 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|300|0
  S2|Ricardo Figueiredo|Lisboa|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
***************
*** 99,105 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|600|270
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 99,105 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|300|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 150,156 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|SELECTION|600|540
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 150,156 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|300|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
