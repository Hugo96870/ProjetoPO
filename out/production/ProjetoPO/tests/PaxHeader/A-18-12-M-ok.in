30 mtime=1606841711.303653164
