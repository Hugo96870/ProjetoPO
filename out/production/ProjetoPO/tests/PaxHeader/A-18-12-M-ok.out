30 mtime=1606841711.304526909
